framework/src/main/java/io/github/kgress/scaffold/webdriver/WebDriverManager.java

```diff
@@ -489,7 +487,7 @@ private RemoteWebDriver configureSauceRemoteBrowser(MutableCapabilities browserO
         // In order to build the URi correctly, pull the username and access key from the desired capabilities bean.
         var username = sauce.getUserName();
         var accessKey = sauce.getAccessKey();
-        var sauceUrl = "@ondemand.saucelabs.com/wd/hub";
+        var sauceUrl = "ondemand.saucelabs.com/wd/hub";
```

Comment:
I'm not sure the username/password are used, I tested a url without the username and password set and it worked fine; I tested a url with the wrong password and it also worked fine. I've made the suggested changes in case someone doesn't set a desired-capabilities.sauce.url though.